mm2src/mm2_main/src/lp_swap/pubkey_banning.rs

```diff
@@ -21,6 +21,7 @@ pub enum BanReason {
 }
 
 pub fn ban_pubkey_on_failed_swap(ctx: &MmArc, pubkey: H256, swap_uuid: &Uuid, event: SwapEvent) {
+    // TODO: db_id
```

Comment:
We also don't need `db_id` for pubkey banning, they are only banned for the current running instance and are cleared on restart.